### PR TITLE
Update insights-agent chart to allow for two installations side by side

### PR DESCRIPTION
**Pull request by katiekeel**
**Why This PR?**
If you try to install a new deployment of `insights-agent` alongside an existing one - as we do installing `fw-telemetry` for clients who already have `insights-agent` present in their cluster - it will fail, because both deployments want to modify the same CRD.

**Fixes**

We need to allow the option to not install the CRD on a deployment.

**Changes**
Changes proposed in this pull request:

* Update the insights-agent OPA `values.yaml` to take an `installCRDs` option, with default of `true`.
* Modify the CRD logic in the insights-agent OPA templates to require the flag to be true to install the CRD.

**Checklist:**

* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] Any new values have been added to the README for the Chart, or helm-docs has been run for the charts that support it.

### CODE REVIEW

File: stable/insights-agent/README.md [hunk 1]
```diff
@@ -79,6 +79,7 @@ Parameter | Description | Default
 `trivy.namespaceBlacklist` | Specifies which namespaces to not scan, takes an array of namespaces for example: `--set trivy.namespaceBlacklist="{kube-system,default}"` | nil
 `opa.role` | Specifies which ClusterRole to grant the OPA agent access to | view
 `opa.additionalAccess` | Specifies additional access to grant the OPA agent. This should contain an array of objects with each having an array of apiGroups, an array of resources, and an array of verbs. Just like a RoleBinding. | null
+`opa.installCRDs` | Specifies whether to install the `customcheckinstances.insights.fairwinds.com` CRD. If you are installing the `insights-agent` chart twice - i.e. once as `insights-agent` and once as `fw-telemetry` - you will want to set this flag to `false` on *one* of the installs, doesn't matter which. | true
```
Comment (lucasreed) [inline stable/insights-agent/README.md]: I don't think we need as much specificity here around fw-telemetry since this is a publicly consumable chart.

**Review by rbren — APPROVED** [review]
Thanks! 🎉

**Review by lucasreed — CHANGES_REQUESTED** [review]
One minor change requested in-line.

Also you'll need to increment the chart version again since pulling in the updated master branch.

**Review by makoscafee — APPROVED** [review]

**Review by lucasreed — APPROVED** [review]
Awesome, thank you! 🎉

**Pull request merged**